src/l500/ac-trigger.cpp

```diff
@@ -670,12 +671,20 @@ namespace ivcam2 {
 
                     auto df = _sf.get_depth_frame();
                     auto irf = _sf.get_infrared_frame();
-                    depth_to_rgb_calibration algo( df, irf, _cf, _pcf, cal_info, cal_regs );
+
+                    depth_to_rgb_calibration algo(df, irf, _cf, _pcf, cal_info, cal_regs, [&]() {return is_processing(); });
```

Comment:
I'd rather you separate the lambda into a variable so it's clear what its function is ('should_continue').
Also, please make the lambda put out a log entry if it returns false or throws.

Reply:
Done.